twilight-gateway/src/shard.rs

```diff
@@ -939,8 +939,18 @@ impl Shard {
     ///
     /// This is primarily useful for sending to other tasks and threads where
     /// the shard won't be available.
-    pub fn sender(&self) -> MessageSender {
-        self.user_channel.sender()
+    pub fn queue(&self) -> MessageQueue {
```

Comment:
I prefer the slightly longer name `message_queue` to avoid confusion with twilight-gateway-queue (`shard.config().queue()`)

Reply:
Resolved as of https://github.com/twilight-rs/twilight/pull/2253/commits/bb21bda6bbc81d9646e5a1d7459d42f3083c59e3.